buttplug/buttplug-device-config/buttplug-device-config.yml

```diff
@@ -2673,6 +2673,19 @@ protocols:
         ScalarCmd:
           - StepRange: [0, 68]
             ActuatorType: Vibrate
+  mizzzee-v2.1:
```

Comment:
Any reason this wasn't just `mizzzee-v3`?

Reply:
I thought v2 is like official protocol version

Reply:
I don't know if they have protocol numbers at all: these after often named by brand and the Nth different protocol we see them using.